mpire/progress_bar.py

```diff
@@ -142,7 +142,7 @@ def _progress_bar_handler(self, tqdm_connection_details: TqdmConnectionDetails,
 
         # In case we're running tqdm in a notebook we need to apply a dirty hack to get progress bars working.
         # Solution adapted from https://github.com/tqdm/tqdm/issues/485#issuecomment-473338308
-        in_notebook = 'IPython' in sys.modules and 'IPKernelApp' in sys.modules['IPython'].get_ipython().config
+        in_notebook = 'IPython' in sys.modules and sys.modules['IPython'].get_ipython() is not None and 'IPKernelApp' in sys.modules['IPython'].get_ipython().config
```

Comment:
It's getting a bit out of hand with the big if statement. Perhaps we should simply let it fail and handle that instead:
```
        try:
            in_notebook = 'IPKernelApp' in sys.modules['IPython'].get_ipython().config
        except (AttributeError, KeyError):
            in_notebook = False
```